Tests/QuickTests/QuickTests/FunctionalTests/ItTests.swift

```diff
@@ -101,6 +101,26 @@ class FunctionalTests_ItSpec: QuickSpec {
 
                 it("should throw an exception with the correct message after this 'it' block executes") {  }
             }
+
+            describe("implicit error handling") {
+                enum ExampleError: Error {
+                    case error
+                }
+
+                func nonThrowingFunc() throws { }
+
+                func throwingFunc() throws {
+                    throw ExampleError.error
+                }
+
+                it("supports calling functions marked as throws") {
+                    try nonThrowingFunc()
+                }
+
+                it("supports calling functions that actually throws") {
```

Comment:
I need a way to test this test as it'll fail once the implementation is correct. I've tried using the exception handling approach, but it doesn't seem like test failures create expections. I think I'd need something like `Nimble` uses to record assertions.